coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -22,23 +22,21 @@
         <template slot="label">
           <div class="certify-stmt">
             I, <b>{{trimmedCertifiedBy || '[Legal Name]'}}</b>, certify that I have relevant knowledge of the
-            association and that I am authorized to make this filing.
+            {{ entityDisplay || 'association' }} and that I am authorized to make this filing.
```

Comment:
Cool! If you plan on using `association` or any other label that is business approved verbiage, in other places you could always make it a constant.

Reply:
I'm OK with the current code. It's legible and easy to understand :)